docs/en/getting-started/get-started-stack.asciidoc

```diff
@@ -6,10 +6,14 @@ Looking for an {stack} ("ELK") guide that shows how to quickly install and confi
 * <<install-elasticsearch,{es}>>
```

Comment:
On line 4:

> Looking for an {stack} ("ELK") guide that shows how to quickly install and configure the {stack}? You're in the right place! You can install the {stack} on a single VM, or even on your laptop:

Do we still want to call this an ELK guide if the L is excluded?

Reply:
I _think_ we do. I know it seems odd, but lots of users still search on "ELK" when they're first getting started. So here's the backstory:

Originally I used "ELK tutorial" in the opening paragraph of this topic because forum users kept searching on "ELK tutorial" and finding a topic on the Digital Ocean website that recommended old Beats settings. These settings were causing major problems. Adding "ELK tutorial" pushed us up quite a bit in the rankings.

It looks like @szabosteve removed "tutorial" from the topic, but that might be OK because the Digital Ocean tutorial isn't completely broken now.

Reply:
@dedemorton 
Yes, it was me back in the DEVO days. Our working group dealt with Getting started/Tutorial/Quickstart definitions and content. As part of that effort, we tried to streamline the wording of the respective pieces. Sometimes it was just to remove a word that defines the function of the piece, like here. I wasn't aware that time the reasoning behind calling it a tutorial or that you are the author of this piece, DeDe. Sorry for the inconvenience and thanks for the heads-up!

Reply:
@szabosteve Yah that makes sense. In this case, I probably should have added a comment to the file to explain why I was doing something that was a little non-standard. :-) I think it makes sense to retain "ELK" here for now...especially since we will be replacing this GS at some point soonish.